components/widgets/inputs/input/input.tsx

```diff
@@ -1,20 +1,24 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
```

Comment:
@julmondragon since you have been playing a lot with this component, would you please pay special attention to these changes since these affect big part of the app. Thanks!